pySDC/implementations/problem_classes/AdvectionEquation_ND_FD.py

```diff
@@ -19,79 +19,79 @@ class advectionNd(ptype):
         dx: distance between two spatial nodes (here: being the same in all dimensions)
     """
 
-    def __init__(self, problem_params, dtype_u=mesh, dtype_f=mesh):
```

Comment:
Why is `dtype_u` and `dtype_f` an argument here in the first place? In the level module this can never be changed.. Is this an inheritance thing? I don't think it makes sense here anyways. I don't know if it can be removed from all problems without any ramifications, though.